Automattic/wp-calypso
# Build: Remove install-if-deps-outdated script

This PR removes the `install-if-deps-outdated` script, as it mostly degrades the developer experience, while not particularly helpful in all cases when there are stale packages or modules. For more context, see: https://github.com/Automattic/wp-calypso/pull/47424#issuecomment-727730874

We're also removing some other scripts that are no longer necessary with this change.

#### Changes proposed in this Pull Request

* Remove the `install-if-deps-outdated` script completely.

#### Testing instructions

* Verify build continues to work correctly with and without a prior `distclean`.
